pkg/registry/apis/provisioning/jobs/migrate/legacy_resources.go

```diff
@@ -70,17 +71,29 @@ func (m *legacyResourcesMigrator) Migrate(ctx context.Context, rw repository.Rea
 	}
 
 	progress.SetMessage(ctx, "migrate folders from SQL")
-	if err := m.folderMigrator.Migrate(ctx, namespace, repositoryResources, progress); err != nil {
-		return fmt.Errorf("migrate folders from SQL: %w", err)
+	if m.clients != nil {
```

Comment:
@MissingRoberto -- I am not sure how you would want this wired up.  for export it injected a wrapper function

Reply:
yeah, I would pass a Export function for testing / decoupling.

Reply:
addressed.